tests/test_flask_admin.py

```diff
@@ -82,7 +84,7 @@ def test_add_distro(self):
                 sess['nickname'] = 'pingou'
                 sess['email'] = 'pingou@pingoured.fr'
 
-            output = c.get('/distro/add', follow_redirects=True)
```

Comment:
Again, just curious: why is this being removed? I guess I'm not sure how `follow_redirects` works in this case. What would the redirect be that we're trying to avoid?